_data/description.yml

```
@@ -4,6 +4,10 @@ aboutDescription: Learn more about the organization, including our main goals, c
 projectsTitle: Projects
 projectsDescription: Our projects cover a wide range of domains, from general functional programming to tooling.
 
+platformsTitle: Platforms
+platformsDescription: Checkout the Typelevel ecosystem by platform.
+jsDescription: Checkout Typelevel on ScalaJS!
```

```suggestion
jsDescription: Checkout Typelevel on Scala.js!
```

Ref: https://github.com/scala-js/scala-js-macrotask-executor/pull/28#discussion_r730727662